interface/themes/style_light.css

```diff
@@ -1286,4 +1286,101 @@ body.admin-layout input[type="button"] {
   border: solid 1px;
   width: 100%;
   border-collapse: collapse;
+}
+
+.attach-input{
+  text-align: right;
+}
+.full-width{
+  width: 100%;
+}
+.radio-pos{
+  margin-left: 10px;
+}
+
+/* ------- Therapy groups styling --------- */
+.group-row{
+  margin-top: 15px;
+}
+.container-group{
+  margin-top: auto;
+}
+.group-error-msg{
+  color: #ff6643;
+}
+.group-success-msg{
+  color: #4c8029;
+}
+
+#component-border{
+  margin-bottom: 20px;
+  border: 1px solid #2672ec;
+  width: 100%;
+  min-height: 150px;
+  padding: 10px;
+}
+#main-component ul{
+}
+.float-right{
+  float: right;
+}
+.rtl .float-right{
+  float: left;
+}
+
+table.dataTable#therapy_groups_list tbody td{
+  padding-left: 20px;
+  padding-right: 20px;
+  font-size: 15px;
+}
+
+.therapy_group #therapy_groups_list_container{
+  margin-top: initial;
+  width: initial;
+}
+
+.therapy_group #therapy_groups_list_container input[type = "text"]{
+  border-color: #ccc;
+}
+
+.therapy_group .delete_btn button, .therapy_group .delete_btn span{
+  background-color: #ef3935;
+  font-size: 15px;
+  padding: 2px 10px;
+}
+
+.therapy_group .delete_btn a{
+  text-decoration: none;
+  color: white;
+}
+
+.therapy_group #therapy_groups_list_container .failed_message{
+  font-size: 15px;
+}
+#updateParticipants{
+  position: relative;
+
+}
+#addParticipant{
+    position: absolute;
+    right: 0px;
+   z-index: 100;
+}
+#add-participant-form{
+  transition: max-height 1s ease-out;
+  max-height: 0;
+  overflow: hidden;
+}
+.showAddForm{
+  max-height: 250px !important;
+}
+.groups-error-msg{
+  color: #ff542d;
+  font-size: 1.3em;
+  padding: 8px;
+  text-align: center;
+}
+
+.therapy_group .event_details{
+  font-size: 14px;
```

Comment:
Repeating above:
"
Recommend making all your changes in 1 stylesheet for now that you are using in the development. Then when code is more mature and nearing completion, can copy the code block over to the other style sheets.
"

If you are ready, then place your stylesheet changes in the other styles.